docs/jwst/outlier_detection/outlier_detection_ifu.rst

```diff
@@ -1,23 +1,29 @@
-.. _outlier_detection_ifu_:
+.. _outlier-detection-ifu:
 
-
-Python Interface to OutlierDetectionIFU: OutlierDetectionIFU()
-=========================================================
+OutlierDetection for IFU Data
+==============================
 
 This module serves as the interface for applying outlier_detection to IFU
  observations, like those taken with NIRSpec and MIRI.  The code implements the
  basic outlier detection algorithm used with HST data, as adapted to JWST IFU
  observations.
 
- Specifically, this routine performs the following operations:
+ Specifically, this routine performs the following operations (modified from the
+ :ref:`outlier-detection-image`):
 
  * Extract parameter settings from input model and merge them with any user-provided values
 
- * Resamples all input images into :py:class:`~jwst.datamodels.IFUCubeModel` observations.
+   - the same set of parameters available to :ref:`outlier-detection-imaging`
+     also applies to this code
+
+ * Resample all input :py:class:`~jwst.datamodels.ImageModel` images into
```

Comment:
As of a couple weeks ago the datamodels got reworked such that the 2D form of IFU images are now an IFUImageModel, instead of a plain ImageModel. So the inputs will now be IFUImageModel.